### DISCUSSION TITLE
Variate flow rate and running time

### DISCUSSION THREAD

**Discussion by HuihuiYangshell** [NatLabRockies/GEOPHIRES-X#277] [Ideas]
Hi ~  
Is there any way for now to use dynamic flow rate with specific running time set, to simulate the results?
Like, If I want to set dictionary table for one day (24h), with different flow rate at different time (hourly), then is there any way to use this table to run simulation within Geophires now? 

Thank you

**Comment by kfbeckers** [comment]
Hi,

Currently, GEOPHIRES does not have the capability to simulate variable flow rate. However, at the recent GEOPHIRES workshop, this was a feature requested by several researchers. We have created an issue (https://github.com/NREL/GEOPHIRES-X/issues/269) and hopefully we can implement this capability soon.

Regards,
Koenraad
